docs/CoreConfig.md

```diff
@@ -3,36 +3,33 @@ Core Configuration Files
 
 The Haraka core reads some configuration files to determine a few actions:
 
-* loglevel
+* log.ini
 
-  Can contain either a number or a string. See the top of logger.js for the
-different levels available.
+Contains settings for log level, timestamps, and format. See the example log.ini file for examples.
+
+* logformat
+
+  Can contain either be LOGFMTDEFAULT for the default log format or
+LOGFMTLOGFMT for the logfmt format.
```

Comment:
I thought this was changes to LOGFMT and DEFAULT?

Reply:
It was, I need to remove this since log.ini was introduced